src/components/CompareResults/loader.ts

```diff
@@ -180,11 +181,28 @@ export async function loader({ request }: { request: Request }) {
     };
   }
 
-  const baseRevFromUrl = url.searchParams.get('baseRev');
+  let baseRevFromUrl = url.searchParams.get('baseRev');
+  let newRevsFromUrl = url.searchParams.getAll('newRev');
+  const baseHashFromUrl = url.searchParams.get('baseHash');
```

Comment:
Where are `baseHashfromUrl` and `newRevsFromUrl` coming from? Not from the inputs in `Compare with Base`, so I'm confused here. I feel these values will always be null and never reach the if condition below to `fetchRevisionFromHash`.

Reply:
Resolved through zoom meeting.